### DISCUSSION TITLE
How to Set Font Rendering in WezTerm to Match Windows Terminal?

### DISCUSSION THREAD

**Discussion by leonasdev** [wezterm/wezterm#3582] [Q&A]
Is there any setting in WezTerm that can replicate the font rendering style of Windows Terminal? I have tried adjusting settings such as `freetype_render_target`, `freetype_load_target`, and `freetype_load_flags`, but no matter how I combine these settings, I cannot achieve the same font rendering style as Windows Terminal.

Although the difference may seem subtle, upon careful observation, the font rendering in Windows Terminal appears smoother. I understand that this may seem like a minor issue, but when displaying a large amount of text, I can clearly perceive the difference.

WezTerm:
![image](https://user-images.githubusercontent.com/39915562/233683477-40bdd8fb-d7da-4f96-9a6a-3a161de1364e.png)

Windows Terminal:
![image](https://user-images.githubusercontent.com/39915562/233683363-7905a710-48d2-4998-991d-cd7e35e7d554.png)

WezTerm:
![image](https://user-images.githubusercontent.com/39915562/233684506-6b1f798c-5bf8-4d72-baf2-110fc0016648.png)

Windows Terminal:
![image](https://user-images.githubusercontent.com/39915562/233684440-504b109c-a843-450e-9323-74128bff8a14.png)

**Comment by LunarLambda** [comment]
I am also struggling to make wezterm's rendering match the rest of my system (on Linux though)

**Reply by LunarLambda** [reply]
Had success with the following:
```
    freetype_load_target = 'Light',
    freetype_render_target = 'HorizontalLcd',
    front_end = 'OpenGL',
```

**Comment by vunhatchuong** [comment]
I also have the same problem, have you found a solution yet? The above doesn't work for me. The difference is easier to see if you look at the indent line and letter `o`. I tried different terminals too, looks like only Windows Terminal can render like this.
Windows Terminal:
![image](https://github.com/wez/wezterm/assets/20581504/f57e69dc-8ebb-44d2-b3e4-6d613f1dbc26)
Wezterm:
![image](https://github.com/wez/wezterm/assets/20581504/6d2ab304-f8b8-4711-ab1a-1a226ceaa31f)

**Comment by Shimoon** [comment]
I was looking for the same thing and couldn't find how to do it.
Example using [LazyGit](https://github.com/jesseduffield/lazygit) to show off special glyphs rendering with [JetBrainsMono Nerd Font](https://www.programmingfonts.org/#jetbrainsmono).

*Windows Terminal :*
![windows_term_rendering](https://github.com/user-attachments/assets/94cfd8e4-2a82-44bd-8e7f-c1e6645c683a)

*WezTerm :*
![wezterm_rendering](https://github.com/user-attachments/assets/e1bc077d-b4eb-4533-a4c2-7a88fe79fdab)

**Reply by Shimoon** [reply]
For anyone that would drop by, I've somewhat successfully matched the Windows Terminal rendering by using those two settings:
- `front_end = "WebGpu"`
- `underline_thickness = "2px"`

I'm using the JetBrainsMono Nerd Font.
